src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputHintSetEventArgs.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+* Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace Tizen.Uix.InputMethod
+{
+    /// <summary>
+    /// This class contains the data related to the InputHintSet event.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public class InputHintSetEventArgs
```

Comment:
shouldn't it be derived from System.EventArgs?
```suggestion
    public class InputHintSetEventArgs : System.EventArgs
```

Reply:
Other EventHandler that previously supports doesn't implement System.EventArgs, but is it necessary to add it?

Reply:
Deriving from System.EventArgs is a recommendation, but does not affect functionality much. Let's keep it as it is for consistency.

Reply:
Ok. I got it.